packages/web-pixel-ui-extensions/README.md

```diff
@@ -0,0 +1,28 @@
+# Web Pixel Extension
+
+Provides an `extend` method to register a Web Pixel Extension.
+
+## Usage
+
+Registering a Web Pixel Extension:
+
+```js
+import {extend} from '@shopify/web-pixel-ui-extensions';
```

Comment:
In a recent conversation with Build, we kind of landed on this convention where things that say `ui` in their name denote a particular kind of extension, in the model of post-purchase/ checkout/ subscription UI extensions. We excluded pixels from that list, opting to instead call it `web pixels` in the CLI selector where you generate a new one. I think including `ui-extensions` in the package name blurs that separation, and I'm feeling like `@shopify/web-pixels` would be better. cc/ @MeredithCastile

Reply:
Keeping `extension` is up to you, I think it works fine either way.